java/basic_app/app/src/main/java/com/appsflyer/onelink/appsflyeronelinkbasicapp/AppsFlyerConstants.java

```diff
@@ -0,0 +1,5 @@
+package com.appsflyer.onelink.appsflyeronelinkbasicapp;
+
+public class AppsFlyerConstants {
```

Comment:
@oded-af please add a comment here and in the main code that this is a devkey of a demo account, and MUST be changed when adding your app.
Same in the iOS.

Thanks